app/Post_Types/Archive_Item/Post_Meta.php

```diff
@@ -158,9 +158,14 @@ public function add_dynamic_fields(){
 		// $type = carbon_get_post_meta( $cptid, Post_Meta::FIELD_TYPE );
 		while( $field_query->have_posts() ) : $field_query->the_post();
```

Comment:
Same story here as above: use `get_posts`, get `ids` only and use `carbon_get_post_meta`

Reply:
Done